Components/Paypal/AddressValidator.php

```diff
@@ -35,6 +35,7 @@ public function __construct(AddressValidatorInterface $innerValidator, DIContain
     public function validate(Address $address)
     {
         if (!$this->container->get('front')->Request()
+            || $this->container->get('front')->Request() === null
```

Comment:
this is already made sure by the query above `!$this->container->get('front')->Request()`